app/Http/Controllers/Api/TalksController.php

```diff
@@ -0,0 +1,22 @@
+<?php namespace Symposium\Http\Controllers\Api;
+
+use Auth;
+use Talk;
+
+class TalksController extends BaseController
+{
+    /**
+     * Display the specified resource.
+     *
+     * @param  string $id
+     * @return Response
+     */
+    public function show($id)
+    {
+        $talk = Talk::where('id', $id)
+                ->where('author_id', Auth::user()->id)
+                ->firstOrFail();
```

Comment:
`$talk = Auth::user()->talks()->findOrFail($id)`

Should ultimately do this permission check earlier though and just trust that it's safe to lookup the ID regardless of the user once passed any ACL. I think I know where I could extract a little package for that :)